### ISSUE TITLE
Share map UX is confusing

### ISSUE THREAD

**Issue by lin-d-hop** [DigitalCommons/land-explorer-front-end#78]
## What is happening
When I click to Share a map via email, after the button click the box just stays open with no indication that the system recognised my action

## What should be happening
After I click Share the box should close, indicating to me that the system has recognised my request.

## Steps to reproduce
1. Share -> Share Invite -> Enter email -> Click Share
2. Nothing seems to happen (I also don't receive the email, see #77)

## Screenshot/GIF
https://images.zenhubusercontent.com/418928297/ea4db569-742f-48b3-b7e7-af750568d181/sharemap.mp4

## Usage Detail
Server: https://testing-lx.com/
OS: Ubuntu 18
Browser: Chrome

## Possible Fix

As a next step I would be looking in Sendgrid to see if the problem pre or post mailserver.

**Comment by King-Mob** [comment]
intention, have it so that if you press share with an email address in the box, the email gets moved to the green phase thing and then it shares it

**Comment by King-Mob** [comment]
@lin-d-hop this is done but in the process I noticed something that makes it slightly more complicated. I noticed that the list of emails you've already shared a map with is supposed to also appear when you click "share by email".

the share button is then supposed to act more like a "sync" button than an "add shared" button. So you're using this interface to edit who can see your map, not just to add new people.

the issue is that the way the app looks at who it's shared with isn't by consulting the database it's by looking at the stupid map json object again. Maybe this should be a new issue, I just don't want to submit a PR that's a regression.
